client/src/components/atoms/select/index.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Import
+ */
+import React from "react";
+import { Select as ChakraSelect } from "@chakra-ui/select";
+
+/*
+ * Types
+ */
+export type Props = {
+  value: string | number;
+  onChange: React.ChangeEventHandler<HTMLSelectElement>;
+  placeholder: string;
+  width?: number;
+};
+
+/*
+ * DOM
+ */
+export const Component: React.FC<Props> = (props) => (
+  <ChakraSelect
+    value={props.value}
+    onChange={props.onChange}
+    placeholder={props.placeholder}
+    width={props.width || 190}
+    fontSize="15px"
+    bg="white"
+    borderColor="white"
+    focusBorderColor="#E6EAEF"
+    _hover={{ bg: "#F3F3F3", borderColor: "#F3F3F3" }}
+  >
+    {props.children}
+  </ChakraSelect>
```

Comment:
このほうがあのProps必要だった！ってなった時に毎回Selectコンポーネントの変更しなくてよさそうかな？って思いました👍このコンポーネントだけじゃなくて全体的にそうした方がよさそうです:)

```suggestion
export type Props = SelectProps & {
  customProps1: string;
  customProps2: number;
}

/*
 * DOM
 */
export const Component: React.FC<Props> = ({ customProps1, customProps2, ...rest }) => (
  <ChakraSelect
    width={props.width || 190}
    fontSize="15px"
    bg="white"
    borderColor="white"
    focusBorderColor="#E6EAEF"
    _hover={{ bg: "#F3F3F3", borderColor: "#F3F3F3" }}
    {...rest}
  >
    {props.children}
  </ChakraSelect>
```

Reply:
ありがとうございます！
たしかに、Chakraの型を利用しておいた方が拡張性あって良いですね！調整します！